README.md

```diff
@@ -72,11 +72,11 @@ apply the snapshot to get the latest state of the chain. The downside here
 is the fact that your node won't have the chain history prior to the snapshot.
 Moreover, you need to trust the source of the snapshot.
 
-Mezo team does not provide any snapshots yet. It will do so for Mezo Matsnet
-testnet in the near future. Until then, you can ask trusted community members
-for a snapshot.
+Mezo team provides snapshots for Mezo Matsnet testnet. Please refer to
+[this runbook](link to 'Runbooks#State sync from snapshot' from this document)
```

Comment:
Looks like the runbook link is not constructed properly:
<img width="1035" alt="image" src="https://github.com/user-attachments/assets/7be555d9-aea5-41a8-a81f-440b45511284" />

Reply:
interesting let me fix it

Reply:
fixed in: 5139b3ff1f4f6f4cf328c4e798ceb474224f8112